core/indigo-core/reaction/src/reaction_json_saver.cpp

```diff
@@ -16,15 +16,43 @@
  * limitations under the License.
  ***************************************************************************/
 
-#include "reaction/reaction_json_saver.h"
+#include <memory>
+#include <vector>
+
+#include <rapidjson/document.h>
+
 #include "base_cpp/output.h"
 #include "molecule/molecule_json_saver.h"
+#include "molecule/query_molecule.h"
 #include "reaction/reaction.h"
+#include "reaction/reaction_json_saver.h"
 
 using namespace indigo;
+using namespace indigo;
+using namespace rapidjson;
 
 IMPL_ERROR(ReactionJsonSaver, "reaction KET saver");
 
+void ReactionJsonSaver::_getBounds(BaseMolecule& mol, Vec2f& min, Vec2f& max, float scale)
```

Comment:
Better to avoid adding names `min` and `max` (literally), because Windows headers historically defined macros `min` and `max`, and we don't use consistent preinclude header.
The same refers to methods `Vec2f::min()` and `Vec2f::max()`, but it's subject of refactoring.